views/winner.erb

```diff
@@ -0,0 +1,19 @@
+<body style="background-color: f5f0e1; color: #1e3d59">
+<div style="font-family: Arial, Helvetica, sans-serif; ">
+  <div style="font-weight: bold; text-align: center; padding: 50px 0 0px 0; display: inline-block; width: 45%; float:left; height:10%; background:#ff6e40; margin:2%">
+   You chose <%= @game.player.choice %>
+  </div>
+  <div style="font-weight: bold; text-align: center; padding: 50px 0 0px 0; display: inline-block; width: 45%; float:right; height:10%; background:#ffc13b; margin:2%">
+    Computer chose <%= @game_choice %><br />
+  </div>
+
+  <div style="text-align: center;  margin: 5% 20% 2% 20%">
+    <div style="margin: 10% 0 5% 0;">
+      <h1><%=@game.winner%></h1>
+    </div>
+    <form action="/">
```

Comment:
This could be an `a href` rather than a form since you are not sending an data with the request